### PR TITLE
PR for navigation and grid

### PR DESCRIPTION
**Pull request by SK02K1**
### Added basic navbar and simplified grids

Screenshot 2022-02-06 at 9 54 41 AM

Screenshot 2022-02-06 at 9 53 48 AM

Screenshot 2022-02-06 at 9 54 03 AM

Screenshot 2022-02-06 at 9 54 22 AM

### CODE REVIEW

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(20rem, 1fr));
+  column-gap: 1rem;
+  row-gap: 2rem;
```
Comment (jeetsdev) [inline components/grid/grid.css]: Can we use `gap: 2rem 1rem` as a shorthand here.

File: components/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.navbar {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  background-color: var(--clr-primary-500);
+  padding: 1rem 1.4rem;
+}
+
+.navbar a {
+  color: #fff;
```
Comment (jeetsdev) [inline components/navigation/navigation.css]: Please use variables here.

**Review by jeetsdev — CHANGES_REQUESTED** [review]

**Review by ayush65 — APPROVED** [review]

File: components/grid/grid.css (hunk 1 above)
Comment (SK02K1) [reply components/grid/grid.css]: I have made those changes, please review

File: components/navigation/navigation.css (hunk 2 above)
Comment (SK02K1) [reply components/navigation/navigation.css]: I have made those changes, please review

**Review by jeetsdev — APPROVED** [review]
Looks good now

**Pull request merged**